src/analytics.ts

```diff
@@ -34,13 +37,16 @@ export function createAnalyticsInstance(options?: Options) {
 
         !user_identity && _rudderstack.identifyEvent(user_identity, { language: user_language || 'en' })
```

Comment:
since you're doing changes, please remove it and make as separate function 🙏 

```suggestion
```

Reply:
optionally we can init `user_identity` as we did for `coreData` and set on `setAttributes` functions and after that reuse it instead of call the `getId`
```suggestion
const identifyEvent =  () => {
    _rudderstack.identifyEvent(getId(), { language: coreData?.user_language || 'en' })
}
```